corehq/apps/registration/views.py

```diff
@@ -158,12 +174,16 @@ def ab(self):
 
     @property
     def page_context(self):
+        prefills = {
+            'email': self.prefilled_email,
```

Comment:
This is okay (just tested). Prelogin's email input is named `e` but HQ's signup form email input is `email`, which is where this is used.